app/main/routes.py

```diff
@@ -92,106 +96,238 @@ def accessibility():
-
-    transferring_body_id = request.args.get("transferring_body_id", None)
-    series_id = request.args.get("series_id", None)
-    consignment_id = request.args.get("consignment_id", None)
-
-    browse_type = "browse"
-    browse_parameters = {}
-    filters = {}
-    sorting_orders = {}
-    breadcrumb_values = {}
     transferring_bodies = []
 
-    if browse_type == "browse":
-        ayr_user = AYRUser(session.get("user_groups"))
-        if ayr_user.is_superuser:
-            for body in Body.query.all():
-                transferring_bodies.append(body.Name)
-        filters = build_filters(request.args)
-        sorting_orders = build_sorting_orders(request.args)
+    ayr_user = AYRUser(session.get("user_groups"))
+    if ayr_user.is_standard_user:
+        return redirect(
+            f"/browse/transferring_body/{ayr_user.transferring_body.BodyId}"
+        )
+    else:
+        # all access user (superuser)
+        for body in Body.query.all():
+            transferring_bodies.append(body.Name)
 
-    if transferring_body_id:
-        browse_type = "transferring_body"
-        browse_parameters["transferring_body_id"] = transferring_body_id
-        breadcrumb_values = {
-            0: {"transferring_body": Body.query.get(transferring_body_id).Name}
-        }
         filters = build_filters(request.args)
         sorting_orders = build_sorting_orders(request.args)
 
-    elif series_id:
-        browse_type = "series"
-        browse_parameters["series_id"] = series_id
-        series = Series.query.get(series_id)
-        body = series.body
-        breadcrumb_values = {
-            0: {"transferring_body_id": body.BodyId},
-            1: {"transferring_body": body.Name},
-            2: {"series": series.Name},
-        }
+        query = build_browse_all_query(
+            filters=filters,
+            sorting_orders=sorting_orders,
+        )
 
-        filters = build_filters(request.args)
-        sorting_orders = build_sorting_orders(request.args)
+        browse_results = query.paginate(page=page, per_page=per_page)
 
-    elif consignment_id:
-        browse_type = "consignment"
-        browse_parameters["consignment_id"] = consignment_id
-
-        consignment = Consignment.query.get(consignment_id)
-        body = consignment.series.body
-        series = consignment.series
-        breadcrumb_values = {
-            0: {"transferring_body_id": body.BodyId},
-            1: {"transferring_body": body.Name},
-            2: {"series_id": series.SeriesId},
-            3: {"series": series.Name},
-            4: {"consignment_reference": consignment.ConsignmentReference},
-        }
-
-        filters = build_browse_consignment_filters(request.args)
-        sorting_orders = build_sorting_orders(request.args)
+        total_records = db.session.query(
+            func.sum(query.subquery().c.records_held)
+        ).scalar()
+        if total_records:
+            num_records_found = total_records
```

Comment:
(same as comment on search PR) Isnt the only option here for query to be a positive integer or 0 so we should be fine by just doing `num_record_found = query.count()`.

Reply:
refactored num_record_found as requested

Reply:
refactored num_record_found as it was before